code/workspaces/infrastructure-api/src/common/nameGenerators.js

```diff
@@ -0,0 +1,29 @@
+const generateDeploymentName = (name, type) => `${type}-${name}`;
```

Comment:
Should this just be `deploymentName` to be aligned with all of the other functions here giving consistent exports at the bottom?

Reply:
That would be the ideal. The reason I haven't done that is because other functions take `deploymentName` as their argument and eslint gets upset that it is already defined. I could add some eslint ignores to those functions if that is the better solution?

Reply:
After offline discussion going to leave as it is but add a comment explaining why it is this way.